docs/run-platform-server/ingestion.mdx

```diff
@@ -34,29 +43,15 @@ stellar-horizon db reingest range <start> <end>
 
 </CodeExample>
 
-_(The command name is a bit of a misnomer: you can use `reingest` both to ingest new ledger data and reingest old data.)_
-
-You can run this process in the background while your Horizon server is up. It will continuously decrement the `history.elder_ledger` in your `/metrics` endpoint until the `<end>` ledger is reached and the backfill is complete. If Horizon receives a request for a ledger it hasn't ingested, it returns a 503 error and clarify that it's `Still Ingesting` (see [below](#some-endpoints-are-not-available-during-state-ingestion)).
-
-#### Deciding on how much history to ingest
+Running any historical range of ingestion requires coordination with the data retention configuration chosen. If you have a temporal limit on history set with `HISTORY_RETENTION_COUNT=<LEDGER_COUNT>` then it makes no sense to ingest any time range that is older as it will get purged from the database almose as soon as it's added.
```

Comment:
May be we can make it more affirmative like "When setting an ingestion time range higher than the temporal limit, the temporal limit takes precedence, and any data ingested beyond that limit will be automatically purged."